src/kotlin_instance.cpp

```diff
@@ -1,81 +1,87 @@
-    } else {
-        return false;
-    }
+//TODO/4.0: Implement new revert methods
+bool KotlinInstance::property_get_revert(const StringName &p_name, Variant &r_ret) const {
+	return true;
 }
 
-void KotlinInstance::get_property_list(List<PropertyInfo>* p_properties) const {
-    kt_class->get_property_list(p_properties);
+bool KotlinInstance::set(const StringName &p_name, const Variant &p_value) {
+	jni::LocalFrame localFrame(1000);
+
+	KtProperty *ktProperty{ kt_class->get_property(p_name) };
+	if (ktProperty) {
+		ktProperty->setCall(binding->kt_object, p_value);
+		return true;
+	} else {
+		return false;
+	}
 }
 
-Variant::Type KotlinInstance::get_property_type(const StringName& p_name, bool* r_is_valid) const {
-    return Variant::VECTOR3;
+bool KotlinInstance::get(const StringName &p_name, Variant &r_ret) const {
+	jni::LocalFrame localFrame(1000);
+
+	KtProperty *ktProperty{ kt_class->get_property(p_name) };
+	if (ktProperty) {
+		ktProperty->callGet(binding->kt_object, r_ret);
+		return true;
+	} else {
+		return false;
+	}
 }
 
-Object* KotlinInstance::get_owner() {
-    return owner;
+void KotlinInstance::get_property_list(List<PropertyInfo> *p_properties) const {
+	kt_class->get_property_list(p_properties);
 }
 
-void KotlinInstance::set_owner(Object *object) {
-    owner = object;
+Variant::Type KotlinInstance::get_property_type(const StringName &p_name, bool *r_is_valid) const {
+	return Variant::VECTOR3;
 }
 
-void KotlinInstance::get_property_state(List<Pair<StringName, Variant>>& state) {
-    ScriptInstance::get_property_state(state);
+void KotlinInstance::get_property_state(List<Pair<StringName, Variant>> &state) {
+	ScriptInstance::get_property_state(state);
 }
 
-void KotlinInstance::get_method_list(List<MethodInfo>* p_list) const {
-    kt_class->get_method_list(p_list);
+void KotlinInstance::get_method_list(List<MethodInfo> *p_list) const {
+	kt_class->get_method_list(p_list);
 }
 
-bool KotlinInstance::has_method(const StringName& p_method) const {
-    return kt_class->get_method(p_method) != nullptr;
+bool KotlinInstance::has_method(const StringName &p_method) const {
+	return kt_class->get_method(p_method) != nullptr;
 }
 
-Variant KotlinInstance::callp(const StringName& p_method, const Variant** p_args, int p_argcount, Callable::CallError& r_error) {
-    jni::LocalFrame local_frame(100);
+Variant KotlinInstance::callp(const StringName &p_method, const Variant **p_args, int p_argcount, Callable::CallError &r_error) {
+	jni::LocalFrame local_frame(100);
 
-    KtFunction* function { kt_class->get_method(p_method) };
-    Variant ret_var;
-    if (function) {
-        function->invoke(this->wrapped_object, p_args, p_argcount, ret_var);
-    } else {
-        r_error.error = Callable::CallError::CALL_ERROR_INVALID_METHOD;
-    }
-    return ret_var;
+	KtFunction *function{ kt_class->get_method(p_method) };
+	Variant ret_var;
+	if (function) {
+		function->invoke(binding->kt_object, p_args, p_argcount, ret_var);
```

Comment:
tab instead of spaces for indenting

Reply:
for a future PR

Reply:
I don't think this should be for a future PR, all files are formatted with spaces.